apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -16,35 +19,69 @@ const defaultArgs: ButtonProps = {
     isFullWidth: false,
     isLoading: false,
     slot: 'This is Lit!',
+    buttonIcons: [],
 };
 
 const buttonStoryMeta: ButtonStoryMeta = {
     title: 'Button',
     component: 'pie-button',
     argTypes: {
         size: {
+            description: 'Set the size of the button.',
             control: 'select',
             options: sizes,
+            defaultValue: {
+                summary: 'medium',
```

Comment:
just curious, what does summary do here?

Reply:
It's basically the only way to add the `defaultValue` in our storybook. Without this syntax (so just assigning a string for instance) the default value doesn't display.

Even stranger than that – `summary` is the only available property you can set in this object 😆